apps/studio/components/interfaces/Organization/BillingSettings/BillingAddress/BillingAddress.tsx

```diff
@@ -87,7 +90,14 @@ const BillingAddress = () => {
         </div>
       </ScaffoldSectionDetail>
       <ScaffoldSectionContent>
-        {!canReadBillingAddress ? (
+        {selectedOrganization?.managed_by !== undefined &&
```

Comment:
we could have this in a component that does this checks with hooks right?

rather than hardcoding "supabase" everywhere.

anyone know what I mean ?

Reply:
Is this so bad? We're showing `PartnerManagedResource` if the org is managed by a partner... the logic seems pretty self-explanatory, no?

Reply:
i think this is fine as long as we have the necessary types for `managed_by` so that comparison values can be type checked

otherwise it feels like an overkill to have an entire hook just for this like
```
export const useIsManagedBySupabase = () => {
  const org = useSelectedOrganization()
  return org.managed_by === 'supabase'
}
```

unless i might be missing something?

Reply:
^ or why not add that check in <PartnerManagedResource />? return null if managed by supabase? (i haven't seen the other files yet so i might be missing something)